### PR TITLE
Fix/year end website refresh issue

### PR DESCRIPTION
**Pull request by timosville**
Card: https://trello.com/c/8EZ5IrYU/1771-year-end-website-refresh-issue

### CODE REVIEW

File: src/_data/cms.js [hunk 1]
```diff
@@ -54,23 +54,41 @@ async function _fetchDevotionRangeFromUrl(url, devotions) {
...
+  const endDayOffset = 3;
+
+  const start = new Date(currentDate.getFullYear(), 0, 0);
+  const diff = currentDate - start;
+  const dayOfYear = Math.floor(diff / (1000 * 60 * 60 * 24));
+
+  let startDay, endDay;
+
+  if (dayOfYear <= 5) {
+    startDay = 1;
```
Comment (JannieT) [inline src/_data/cms.js:68]: How about avoiding all those else statements with early returns like this:

```suggestion
    return { 1, 8 };
```

**Review by JannieT — CHANGES_REQUESTED** [review]
It looks like it might work while we are all on holiday. How about using Playwright's unit testing features to put some more confidence into it? To isolate it for testing, you can either export this function from an assets/js/utils.js module or, since it is only one function, duplicate it straight into the Playwright spec file, test till we are confident and then paste it back into the cms.js file.

File: src/_data/cms.js (hunk 1 above)
Comment (timosville) [reply src/_data/cms.js]: Nice suggestion.

**Review by JannieT — APPROVED** [review]
well done!

**Pull request merged**